### DISCUSSION TITLE
Build Trust Without Container Trust

### DISCUSSION THREAD

**Discussion by DJDVS1** [GriefPrevention/GriefPrevention#2300] [Ideas]
One of my server members was building a farm and got a random person to help him but that person took loot from his farm without him knowing cuz he had build trust. And yes, theres subclaiming but this seems more useful

**Comment by DJDVS1** [comment]
The player would also like to know if you like Jazz

**Comment by Jikoo** [comment]
An addon could do this. As far as base GP is concerned, breaking blocks but not accessing inventory contents is a little illogical - the person could just have broken a chest to loot it. Sounds like your player learned the hard way why you don't grant trust to your valuables to random people :\
